x-pack/docs/en/security/securing-communications/tls-versions-jdk.asciidoc

```diff
@@ -0,0 +1,144 @@
+[role="xpack"]
+[[jdk-tls-versions]]
+=== Supported SSL/TLS versions by JDK version
+
+{es} relies on your JDK's implementation of SSL and TLS.
+
+Different JDK versions support different versions of SSL, and this may affect how {es} operates.
+
+`SSLv3`::
+  SSL v3 is supported on all {es} <<jvm-version,compatible JDKs>> but is disabled by default.
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1`::
+  TLS v1.0 is supported on all {es} <<jvm-version,compatible JDKs>>.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.0 by default.
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.1`::
+  TLS v1.1 is supported on all {es} <<jvm-version,compatible JDKs>>.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.1 by default.
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.2`::
+  TLS v1.2 is supported on all {es} <<jvm-version,compatible JDKs>>.
+  It is enabled by default on all JDKs that are supported by {es}, including the bundled JDK.
+
+`TLSv1.3`::
+  TLS v1.3 is supported on all recent {es} <<jvm-version,compatible JDKs>>.
```

Comment:
I was intentionally vague, because an in depth answer would take several paragraphs :)

What we mean is:
- TLS 1.3 is supported on JDK15 and JDK16.
- TLS 1.3 is supported on most releases of JDK11 (maybe all the GA ones, I'd need to check the history)
- TLS 1.3 is supported on some JDK8 builds, including the most recent release of each JDK8 distribution that ES supports
- However TLS 1.3 is not supported on _every_ JDK8 build, and ES supports running on older JDK8 builds that don't have TLS 1.3. We would encourage customers to upgrade to a JDK version that has TLS 1.3 (if only because newer JDKs are typically better), but it's not true to say "Every compatible JDK supports TLS 1.3"

If we can get that down to a couple of clear sentences that would be awesome.